modules/autocomplete/main/index.coffee

```diff
@@ -131,7 +131,7 @@ buildAutoComplete = (searchTerm, fromCallback, loading) ->
     if not filteredData?
       message.text = @options.loadingMessage
     else if searchTerm.length < @options.minLength
-      message.text = @options.minLengthMessage.replace('$minLength', @options.minLength)
+      message.text = @options.pleaseEnterMinCharactersMessage.replace('$minLength', @options.minLength)
```

Comment:
Possibly - although I think the option was added in 1.4.0 so it would be a bugfix